DNN Platform/Library/Security/Membership/AspNetMembershipProvider.cs

```diff
@@ -1767,10 +1767,7 @@ public override void UpdateUsersOnline(Hashtable userList)
             //Global Data Store but not in the Local DataStore ie. A shared Global Data Store
 
             //Initialise Login Status to Failure
-            loginStatus = UserLoginStatus.LOGIN_FAILURE;
-
-            DataCache.ClearUserCache(portalId, username);
-            DataCache.ClearCache(GetCacheKey(username));
+            loginStatus = UserLoginStatus.LOGIN_FAILURE;       
```

Comment:
By removing this call at the beginning it is possible that when beginning the login call you will get a cached user object, which might not have been properly invalidated, thus allowing the user to be identified as a non-deleted user.  I don't believe it would be safe to remove this call here?  Can you confirm the validation that this did not cause issues?